core-web/libs/template-builder/src/lib/components/template-builder/components/remove-confirm-dialog/remove-confirm-dialog.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angular/core';
+
+import { ConfirmationService } from 'primeng/api';
+import { ButtonModule } from 'primeng/button';
+import { ConfirmPopupModule } from 'primeng/confirmpopup';
+
+@Component({
+    selector: 'dotcms-remove-confirm-dialog',
+    templateUrl: './remove-confirm-dialog.component.html',
+    styleUrls: ['./remove-confirm-dialog.component.scss'],
+    standalone: true,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    imports: [ConfirmPopupModule, ButtonModule],
+    providers: [ConfirmationService]
+})
+export class RemoveConfirmDialogComponent {
+    @Output() confirmEvent: EventEmitter<void> = new EventEmitter();
```

Comment:
Since it's only emitted when the user confirms the delete action.

```suggestion
    @Output() deleteConfirmed: EventEmitter<void> = new EventEmitter();
```